### DISCUSSION TITLE
Always show Mirador collection menu

### DISCUSSION THREAD

**Discussion by medau9** [ProjectMirador/mirador#3580] [Looking for examples]
I have a question towards the community. I am trying to always have the collection menu showed, the moment the collection is opened.
What i mean is that whenever I want to see the full list, i first have to go to the Index, on the left, then click on the volume and then a menu is opened on the right side. Is there a possibility for this menu to always be fixed on the right side, so the users have it easier to find a manifest on the collection. 

Thank you in advance!

**Comment by marlo-longley** [comment]
Sorry this got buried in Discussion. Converted to Issue: https://github.com/ProjectMirador/mirador/issues/4521
